src/index.js

```diff
@@ -1,11 +1,13 @@
-import app from './app';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App.js';
```

Comment:
```suggestion
import App from './app.js';
```

Reply:
Changing the case of the App.js file throws an eslint error.
Something about not being the convention with react, where all component names must start with an uppercase letter

Reply:
Finally went with the lowercase naming convention for the app file